src/redux/missions/missionsSlice.js

```diff
@@ -19,6 +19,16 @@ const initialState = {
 const missionsSlice = createSlice({
   name: 'missions',
   initialState,
+  reducers: {
+    joinMission: (state, action) => {
+      const newState = state.missions.forEach((element) => {
+        if (element.mission_id === action.payload) {
+          element.reserved = !element.reserved;
```

Comment:
Great logic!

Reply:
Thanks a lot for you comments, @Alejandro-Bernal-M, I think you're going to be an excellent code reviewer 👏🏼